index.js

```diff
@@ -1,82 +1,136 @@
-let repoList = []
+let repoList = [];
+const SEARCH_TERM = "-web-";
 
 function handleGetLabs() {
   if (getToken() && getUsername()) {
-    getRepoList(1)
-    addStatus("Getting Repos")
+    getRepoList(1);
+    addStatus("Getting Repos");
   } else {
-    console.log("Please provide your GitHub token and username.\nSee https://github.com/achasveachas/GitHub-cleaner/blob/master/README.md for details.")
+    console.log(
```

Comment:
I'm thinking now that giving the error in the console is probably bad UX. The error should probably be displayed to the user on the screen or through an alert pop-up.

This is not related to your code, so I'm not expecting you to make this change. Feel free to ignore and I'll see if I can find time to add that at a later time :)